.github/workflows/cli-responsible-ai-cli-responsibleaidashboard-housing-classification-cli-responsibleaidashboard-housing-classification.yml

```diff
@@ -6,7 +11,6 @@ on:
   pull_request:
     branches:
       - main
```

Comment:
Still not sure if we should update these workflows... I am not sure who the owner of these workflows is, but maybe check with them?

Also looks like we need to run the readme.py only if adding new files or changing descriptions... since we just renamed an existing file maybe this doesn't apply to us afaiu...?

```
Also if adding new examples or changing existing descriptions, run the readme.py script in the respective subdirectory to generate the README.md file with the table of examples:
```

Reply:
In the previous PR [https://github.com/Azure/azureml-examples/pull/1879], we had the same doubt. 
@harneetvirk recommended to keep the changes post running cli/readme.py. [reference - https://github.com/Azure/azureml-examples/pull/1879#discussion_r1024303236]

Reply:
Sounds good